packaging/osx/buildpackage.sh

```diff
@@ -12,9 +14,12 @@ if [ -e "OpenRA.app" ]; then
     exit 2
 fi
 
+wget https://github.com/OpenRA/OpenRALauncherOSX/releases/download/${LAUNCHER_TAG}/launcher.zip || exit 3
```

Comment:
You told me in https://github.com/OpenRA/OpenRA/pull/7953 to use curl instead as that is installed by default on Mac. Shall I change it in https://github.com/OpenRA/OpenRA/pull/7953, too?